mlir/test/Conversion/NVGPUToNVVM/nvgpu-to-nvvm.mlir

```diff
@@ -1339,3 +1339,19 @@ module attributes {transform.with_named_sequence} {
     transform.yield
   }
 }
+
+// CHECK-LABEL: @rcp_approx_ftz_f32
+// CHECK-SAME:  %[[IN:.*]]: vector<32x16xf32>
+func.func @rcp_approx_ftz_f32(
```

Comment:
Would be great if you also add a failing test, so we check the verifier

Reply:
Done.